test/EnumTest.php

```diff
@@ -53,7 +52,7 @@ public function it_tests_equals()
         $diet_style = DietStyle::OMNIVORE();
         $diet_style2 = DietStyle::OMNIVORE();
 
-        $this->assertTrue($diet_style->equals($diet_style2));
-        $this->assertTrue($diet_style2->equals($diet_style));
+        self::assertTrue($diet_style->equals($diet_style2));
```

Comment:
Warum hier ein statischer Aufruf?

Reply:
Ich dachte ich hätte hier alle $this-> in self:: umgewandelt. muss ich nochmal schauen.
Der Grund für die Umstellung ist. PHPStorm hat sich beschwert und PHPUnit definiert alle assert Funktionen als public static function also beschwert sich PHPStorm richtiger weise. 

Ich muss aber nochmal schauen ob das an der Version von PHPUnit liegt und va. alle aufrufe gleich machen.